plugins/modules/monitoring/alerta_customer.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Christian Wollinger <@cwollinger>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: alerta_customer
+short_description: Manage customers in Alerta via the REST API
```

Comment:
I'd keep this one shorter and add the part with the REST API in the main description:
```suggestion
short_description: Manage customers in Alerta
```

Reply:
Changed the description